### ISSUE TITLE
Interactive Transcript Tab Appearing, but it is Also Empty

### ISSUE THREAD

**Issue by owlboy** [oel-mediateam/sbplus_v3#62]
I have found an example of where an interactive transcript is appearing in a presentation, but it is not showing any text.

Also, the first time I switched to the tab, it did not clear the content from the first tab from the screen. 

**Example Presentation:** https://media.uwex.edu/content/ds/ds740/ds740_lesson3_presentation2/

**Comment by Lin87** [comment]
I am not seeing it on my end. Try on a new browser and/or clearing cache. And see if the interactive transcript is enabled in the settings.

**Comment by owlboy** [comment]
I cleared cache, and the interactive transcript is not disabled. 

Either way, the other issues still stand.

https://youtu.be/elEcIUeE9xE

**Comment by Lin87** [comment]
I tested it on Firefox, Chrome, Safari, (on both MacOS and Windows 10) and IE/Edge. I did not run into the issue where the interactive transcript is showing up. It only shows up when I enabled it through the settings. So, my guess is you enabled interactive transcript sometime ago. Clearing cache does not clear localstorage/webstorage in Safari. You would to remove website data through the privacy tab under Safari preferences. I have a Chrome extension that clear webstorage and cache with a click of a button.

As for the other issue, it is related to the interactive transcript. There was an error with parsing the caption file for the interactive transcript, so the script stopped and did not clear the content from previously viewed tab. The error was caused by a time code range in the WebVTT file with no caption text in that range. It threw the parser off. I added a try catch statement in the code, so when there is an error with interactive transcript it will display a message about some complications with the caption file, but it will not break the script, and previously viewed tab content will be cleared on tab change. Also, interactive transcript is an alpha feature that I will be restructuring/re-coding soon.
